.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "Dev",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": { "VARIANT": "18" }
+	},
+
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"dbaeumer.vscode-eslint",
+				"ms-vscode.powershell",
+				"eg2.vscode-npm-script",
+				"mikestead.dotenv",
+				"christian-kohler.npm-intellisense"
+			]
+		}
+	},
+	"forwardPorts": [1521],
```

Comment:
Should this forward 5672 15672?

Reply:
Never mind. I was assuming we also do optimize for Linux like I built into RavenDB setup

https://github.com/Particular/setup-ravendb-action/blob/main/setup.ps1#L21-L45

FYI this makes the linux builds WAY faster than running on ACI too.

Reply:
I think that wouldd be a good addition but I'll leave that to a separate PR since we're not using local containers at all right now so there should be no negative impact for this RN.